src/sentry/static/sentry/app/views/projects/projectContext.jsx

```diff
@@ -95,6 +95,17 @@ const ProjectContext = createReactClass({
       this.fetchData();
     }
 
+    // Project list has changed. Likely indicating that a new project has been
+    // added. Re-fetch project details in case that the new project is the active
+    // project.
+    //
+    // For now, only compare lengths. It is possible that proeject slugs within
```

Comment:
```suggestion
    // For now, only compare lengths. It is possible that project slugs within
```

Reply:
damn, my typos are terrible. Never learned to proof read 🥉